PosProject/src/main/webapp/WEB-INF/pages/users.jsp

```diff
@@ -0,0 +1,41 @@
+<%-- 
+    Document   : users
+    Created on : 29.12.2021, 18:47:50
+    Author     : stupa
+--%>
+
+<%@page contentType="text/html" pageEncoding="UTF-8"%>
+<!DOCTYPE html>
+<%@taglib prefix="t" tagdir="/WEB-INF/tags" %>
+<%@taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+
+<t:pageTemplate pageTitle="Users">
+    <h1>Users</h1>
+    <form method="POST" action="${pageContext.request.contextPath}/Users">
+        <%-- <c:if test="${pageContext.request.isUserInRole('AdminRole')}">
```

Comment:
Why do we need to comment all the button? For removing the role we only need to comment or delete the it condition